docs/guides/advanced.rst

```diff
@@ -151,3 +151,59 @@ differently. Put something like this in your configuration file::
 
 Used together, flexible attributes and path format conditions let you sort
 your music by any criteria you can imagine.
+
+
+Automatically add new music to your library
+-------------------------------------------
+
+As a command-line tool, beets is perfect for automated operation via a cron job
+or the like. To use it this way, you might want to use these options in your
+`config file`_::
+
+    [beets]
+    import_incremental: yes
+    import_quiet: yes
+    import_log: /path/to/log.txt
+
+The ``import_incremental`` option will skip importing any directories that have
+been imported in the past.
+``import_quiet`` avoids asking you any questions (since this will be run
+automatically, no input is possible).
+You might also want to use the ``import_quiet_fallback`` options to configure
+what should happen when no near-perfect match is found -- this option depends
+on your level of paranoia.
+Finally, ``import_log`` will make beets record its decisions so you can come
+back later and see what you need to handle manually.
+
+The last step is to set up cron or some other automation system to run
+``beet import /path/to/incoming/music``.
+
+.. _config file: http://beets.readthedocs.org/page/reference/config.html
+
+
+Useful reports
+--------------
+
+Since beets has a quite powerful query tool, this list contains some useful and
+powerful queries to run on your library.
+
+* See a list of all albums which have files which are 128 bit rate::
+
+      beet list bitrate:128000
+
+* See a list of all albums with the tracks listed in order of bit rate::
+
+      beet ls -f '$bitrate $artist - $title' | sort -n
+
+* See a list of all albums with the tracks listed in order of sample rate::
+
+      beet ls -f '$samplerate $artist - $title' | sort -n
+
+* See a list of albums and their formats::
+
+      beet ls -f '$albumartist $album $format' | sort | uniq
+
+  Note that ``beet ls --album -f '... $format'`` doesn't do what you want,
+  because there is no notion of an album format.
```

Comment:
Or, put in a slightly more beets-like way: "…because $format is an album-level field, not an item-level one."